### PR TITLE
Added Percentage Price Oscillator

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

**Comment by iblis17** [comment]
Is there any reference implementation that I can compare the outputs?

File: src/momentum.jl [hunk 1]
```diff
@@ -243,3 +243,37 @@ function stochasticoscillator(ohlc::TimeArray, n::Integer=14, fast_d::Integer=3,
...
+**Return**:
+
+`TimeArray` with 3 columns `["ppo", "percentdif", "signal"]`.
+
+If the input is a multi-column `TimeArray`, the new column names will be
+`["A_macd", "B_macd", "A_dif", "B_dif", "A_signal", "B_signal"]`.
+
+"""
+
+function ppo(ta::TimeArray{T,N},fast::Int=12, slow::Int=26, signal::Int=9;wilder::Bool=false)
```
Comment (iblis17) [inline src/momentum.jl]: `function ppo(ta::TimeArray, ...)` is enough, you do not need the parameter of `TimeArray` in this case.

**Comment by Vaibhavdixit02** [comment]
https://github.com/askmike/gekko/blob/develop/strategies/indicators/PPO.js I'll try using this to compare the results? @iblis17

**Comment by iblis17** [comment]
Okay, please add some test cases if possible.
For example, there is an implementation of RSI in TTR (R's package).
We can just copy the output form R as our testing answer.
e.g.
https://github.com/JuliaQuant/MarketTechnicals.jl/blob/master/test/momentum.jl#L14

**Pull request closed without merging**